game/gothic.cpp

```diff
@@ -595,6 +598,14 @@ phoenix::script Gothic::loadPhoenixScriptCode(std::string_view datFile) {
   return phoenix::script::parse(buf);
   }
 
+int Gothic::settingsSystemPackGetI(std::string_view sec, std::string_view name) {
```

Comment:
Only call path is `settingsSystemPackGetI("PARAMETERS","HideFocus")`
Should be cached into bool, similar to `frate`